code/modules/lighting/lighting_turf.dm

```diff
@@ -1,90 +1,42 @@
+//Estimates the light power based on the alpha of the light and the range.
+//Assumes a linear fallout at (0, alpha/255) to (range, 0)
+#define LIGHT_POWER_ESTIMATION(alpha, range, distance) max((alpha * (range - distance)) / (255 * range), 0)
+
 /turf
-	var/dynamic_lighting = TRUE
-	luminosity           = 1
 
-	var/tmp/lighting_corners_initialised = FALSE
+	luminosity           = 1
 
-	var/tmp/list/datum/light_source/affecting_lights       // List of light sources affecting this turf.
-	var/tmp/atom/movable/lighting_object/lighting_object // Our lighting object.
-	var/tmp/list/datum/lighting_corner/corners
+	var/tmp/list/lights_affecting
```

Comment:
should be a typed list if you're using `as()` on it

Reply:
Thanks, on it